Jenkinsfile

```diff
@@ -79,24 +84,39 @@ pipeline {
     stage ("Initial") {
       steps {
         echo GREEN("${getCauseDescription(currentBuild)}")
+
+        script {
+          // If build is triggered by PR, use PR branch, otherwise use master
+          if (IS_TRIGGERED_FOR_PR) {
+            GIT_BRANCH = env.sha1
+          } else {
+            GIT_BRANCH = 'master'
+          }
+        }
+
+        echo GREEN("Branch: $GIT_BRANCH")
+
+        // Print the build envvars
+        echo sh(returnStdout: true, script: 'env')
+      }
+    }
+
+    stage ("Prepare (triggered by PR)") {
+      when { expression { IS_TRIGGERED_FOR_PR } }
```

Comment:
for => by

you use by in all other cases.

Reply:
Fixed